cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkAuthenticationExceptionHandler.java

```diff
@@ -55,10 +65,49 @@
         DEFAULT_ERROR_LIST.add(OneTimePasswordRequiredException.class);
     }
 
+    static {
```

Comment:
three exceptions that will trigger login throttle:
- wrong username: `AccountNotFoundException`
- wrong password: `FailedLoginException`
- wrong one time password: `OneTimePasswordFailedLoginException`